### PR TITLE
fix: improve html

### PR DESCRIPTION
**Pull request by jbergstroem**
Few things the validator wasn't happy about

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,9 +1,10 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 
 <head>
-  <meta name="viewport" content="width=device-width, initial-scale=1" />
   <title>Topsort Blocks Demo</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="favicon" href="/favicon.ico">
```
Comment (PabloReszczynski) [inline index.html:7]: We don't have a favicon

File: index.html [hunk 2]
```diff
@@ -162,7 +163,7 @@
       <!-- REUSABLE ELEMENTS -->
       <div hidden id="product-proto" class="product">
         <div class="product-img-name">
-          <img class="product-img" />
+          <img class="product-img"">
```
Comment (PabloReszczynski) [inline index.html:166]: what does that quote at the end mean

File: index.html (hunk 1 above)
Comment (jbergstroem) [reply index.html:7]: https://github.com/Topsort/building-blocks/blob/main/public/favicon.ico

File: index.html (hunk 2 above)
Comment (jbergstroem) [reply index.html:166]: I'll ask my editor </excuse>

**Review by sofiavalenzuelapugh — APPROVED** [review]

**Pull request merged**